float32 trajduration
trajectory_msgs/JointTrajectoryPoint pregraspConfig
trajectory_msgs/JointTrajectoryPoint graspConfig
---
